website/tools.rst

```diff
@@ -14,6 +14,12 @@ operations, consider `oiiotool
 Swiss Army Knife of `OpenImageIO
 <https://sites.google.com/site/openimageio/home>`_
 
+To build a tool in a git bash window:
```

Comment:
This page isn't intended to discuss building the tools, just how to use them, so I don't think this is appropriate here.

Reply:
I also think this is where the error is that causes the build to fail, see the "Website/Website" link below.

Reply:
Where would I learn how to build the writer app? Should we just refer over to the technical details Hello World example? From there, we need where to put the files and what to install and what to run.

Reply:
The tools get built as a component of the full project build as long as `OPENEXR_BUILD_TOOLS` is `ON`, which is is by default. But I guess that's not entirely obvious, so worth mentioning in the other parts of the documentation.

Reply:
But the instructions below won't work because the tool directory is not set up for an independent build. The more appropriate thing to say here is something like the previous comment, the tools are built as a component of the overall project build when `OPENEXR_BUILD_TOOLS` is `ON`.

And the indentation is off, so it's causing the website build to fail as well.